src/components/Dropdown/Dropdown.js

```diff
@@ -115,37 +130,39 @@ class Dropdown extends Component {
       >
         {isOpen
           ? <EventListener target={window} onClick={this._onWindowClick} />
-          : null
-        }
+          : null}
         <div onClick={computedToggleFunc} style={computedTriggerStyle}>
           {triggerNode}
         </div>
         {arrow && isOpen ? ArrowThing : null}
-        {isOpen
-          ? <div style={computedChildrenStyle}>{children}</div>
-          : null
-        }
+        {isOpen ? <div style={computedChildrenStyle}>{children}</div> : null}
       </div>
     )
   }
   _onWindowClick = () => {
-    const {closeOnClick} = this.props
-    const {mouseIsOver, stillPropagating} = this.state
-    const {isOpen, closeDropdown} = this._getControls()
+    const { closeOnClick } = this.props
+    const { mouseIsOver, stillPropagating } = this.state
+    const { isOpen, closeDropdown } = this._getControls()
     if (closeOnClick && stillPropagating) {
-      this.setState({stillPropagating: false}) // <-- bug fix for the synthetic event propagation... wow
+      this.setState({ stillPropagating: false }) // <-- bug fix for the synthetic event propagation... wow
     } else if (!mouseIsOver || closeOnClick) {
       closeDropdown()
     }
   }
   _getControls = () => {
-    const {openDropdown, closeDropdown, isOpen} = this[this._hasExternalControls() ? 'props' : 'state']
-    return {openDropdown, isOpen, closeDropdown}
+    const { openDropdown, closeDropdown, isOpen } = this[
+      this._hasExternalControls() ? 'props' : 'state'
+    ]
+    return { openDropdown, isOpen, closeDropdown }
   }
   _hasExternalControls = () => {
     // only if we have full controll from outside can use use props
-    const {openDropdown, closeDropdown, isOpen} = this.props
-    return (openDropdown !== undefined && closeDropdown !== undefined && isOpen !== undefined)
+    const { openDropdown, closeDropdown, isOpen } = this.props
+    return (
+      openDropdown !== undefined &&
+      closeDropdown !== undefined &&
+      isOpen !== undefined
+    )
   }
 }
```

Comment:
Oh, maybe I branched out before it was on master, no problem I can checkout just the dropdown folder from there.